lms/templates/admin/instance.new.html.jinja2

```diff
@@ -2,25 +2,27 @@
 
 {% extends "lms:templates/admin/base.html.jinja2" %}
 
-
 {% block header %}
 New application instance
 {% endblock %}
 
 {% block content %}
 
-<form method="POST" action="{{ request.route_url("admin.registration.new.instance", id_=lti_registration.id) }}">
-    <input type="hidden" name="csrf_token" value="{{get_csrf_token()}}">
-    <fieldset class="box mt-6">
-        <legend class="label has-text-centered">Registration</legend>
+<fieldset class="box mt-6">
+    <legend class="label has-text-centered">Registration</legend>
 
-        {{ macros.registration_preview(request, lti_registration) }}
-    </fieldset>
+    {{ macros.registration_preview(request, lti_registration) }}
+</fieldset>
 
+<form method="POST" action="{{ request.route_url("admin.instance.new") }}">
+    <input type="hidden" name="csrf_token" value="{{ get_csrf_token() }}">
+    <input type="hidden" name="lti_registration_id" value="{{ lti_registration.id }}">
```

Comment:
We now persist the `lti_registration_id` as a param. This make it possible for it to be optional, and also makes it really easy to pass the params back to ourselves, by passing them all.